crates/hc_sandbox/tests/cli.rs

```diff
@@ -31,7 +31,7 @@ async fn call_app_interface(port: u16) {
     tracing::debug!(calling_app_interface = ?port);
     let (mut app_tx, _) = websocket_client_by_port(port)
         .await
-        .expect(&format!("Failed to get port {}", port));
+        .unwrap_or_else(|_| panic!("Failed to get port {}", port));
```

Comment:
Is this what clippy wants?

Reply:
probably. this is part of the code changes required for the 1.71.1 bump which @neonphog backported